templates/long.yaml

```diff
@@ -5,9 +5,32 @@
 
 - name: Test Project - With Folders
   type: Project
+  acl:
+  - principal:
+      type: Team
+      id: 3379097
+      name: Sage Bio DREAM Curators
+    principal_id: 3379097
+    access_type:
+    - READ
+    - DOWNLOAD
+    - UPDATE
+    - CREATE
+    - DELETE
+    - CHANGE_SETTINGS
+    - MODERATE
+    - CHANGE_PERMISSIONS
```

Comment:
I like using these atomic access types in the long template because that's how Synapse [models](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/ResourceAccess.html) ACLs. We can support the web client shortcuts (_e.g._ `edit`, `admin`, etc.) in the short template. 

That said, the template author can always use YAML anchors/aliases in the long template. I agree that we shouldn't use them in this minimal template just to keep things simple.

Reply:
I like the idea of supporting the web client shortcuts in the short version despite the added work.  For the long template, we should keep permissions this way, following the API.  If I was to extract a project structure from a Synapse Project, this is the representation I expect to see.